kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/ClusterService.java

```diff
@@ -247,8 +244,15 @@ public TopicsResponse getTopics(String name, Optional<Integer> page,
         .orElseThrow(ClusterNotFoundException::new);
     var topic = getTopicDetails(clusterName, topicName)
         .orElseThrow(TopicNotFoundException::new);
-    return kafkaService.deleteTopic(cluster, topic.getName())
-        .doOnNext(t -> updateCluster(topicName, clusterName, cluster));
+    return brokerService.getBrokerConfigMap(cluster, cluster.getBrokers().get(0))
```

Comment:
ideally we should check configs on controller node, not first

Reply:
Btw you already have this features in cluster feature list, as for me no sense to recheck it.

Reply:
It seems to me that we need to have a check not only on the client but also on the server